server/src/main/java/org/eclipse/lsp/cobol/core/preprocessor/delegates/copybooks/analysis/CopybookName.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2022 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *    Broadcom, Inc. - initial API and implementation
+ *
+ */
+
+package org.eclipse.lsp.cobol.core.preprocessor.delegates.copybooks.analysis;
+
+import lombok.Value;
+
+/**
+ * This value class represents a copybook name with dialect info
+ * context
+ */
+@Value
+public class CopybookName {
+  String name;
+  String dialectType;
+
+  /**
+   * Extract a copybook name from processing copybook name
+   * @param processingName a processing name
+   * @return a copybook name
+   */
+  public static String extractName(String processingName) {
```

Comment:
Maybe it makes sense to use the default dialect type for tests instead of normalizing names?

Reply:
So, my concern is that we can't test 2 different dialect copybooks with the same name in that case in the future